pkg/ingester/tokens

```diff
@@ -0,0 +1 @@
+�\�
```

Comment:
Is this file expected to be committed?

Reply:
Definitely not. Forgot to push a follow-up cleanup PR.